boards/ct/ctcc/doc/index.rst

```diff
@@ -117,55 +174,113 @@ logs on emulated USB port.
 available for signing your binary for MCUboot as described on :ref:`west-sign`.
 
-Next, do the **one-time setup** to flash MCUboot with DFU support.
-We'll assume you've cloned the `MCUboot`_ as a submodule when initializing
-Zephyr repositories using :ref:`west` tool.
+#. Compile MCUboot as a Zephyr application with ``mcumgr`` support.
+
+.. tabs::
+
+   .. group-tab:: nRF52840
+
+      Use following configs to build MCUboot:
+
+      .. code-block:: cfg
+
+         CONFIG_UART_CONSOLE=n \
+         CONFIG_MCUBOOT_SERIAL=y \
+         CONFIG_BOOT_SERIAL_ENTRANCE_GPIO=n \
+         CONFIG_BOOT_USB_DFU_WAIT=n \
+         CONFIG_BOOT_SERIAL_CDC_ACM=y \
+         CONFIG_BOOT_SERIAL_WAIT_FOR_DFU=y \
+         CONFIG_BOOT_SERIAL_NO_APPLICATION=y \
+         CONFIG_MULTITHREADING=y \
+         CONFIG_USB_DEVICE_STACK=y \
+         CONFIG_BOOT_SERIAL_WAIT_FOR_DFU_TIMEOUT=5000 \
+         CONFIG_USB_DEVICE_REMOTE_WAKEUP=n
+
+      To build the MCUboot:
+
+      .. code-block:: console
+
+         west build \
+            -b ctcc/nrf52832 \
+            mcuboot/boot/zephyr
+
+   .. group-tab:: nRF9161
+
+      Use following configs to build MCUboot:
 
-#. Compile MCUboot as a Zephyr application with DFU support.
+      .. code-block:: cfg
 
-   .. zephyr-app-commands::
-      :app: mcuboot/boot/zephyr
-      :board: ctcc/nrf52840
-      :build-dir: mcuboot
-      :goals: build
-      :gen-args: -DCONFIG_BOOT_USB_DFU_WAIT=y
+         CONFIG_BOOT_SERIAL_UART=y \
+         CONFIG_UART_CONSOLE=n \
+         CONFIG_MCUBOOT_SERIAL=y \
+         CONFIG_BOOT_SERIAL_ENTRANCE_GPIO=n \
+         CONFIG_BOOT_USB_DFU_WAIT=n \
+         CONFIG_BOOT_SERIAL_WAIT_FOR_DFU=y \
+         CONFIG_BOOT_SERIAL_NO_APPLICATION=y \
+         CONFIG_BOOT_SERIAL_WAIT_FOR_DFU_TIMEOUT=5000
+
+      To build the MCUboot:
+
+      .. code-block:: console
+
+         west build \
+            -b ctcc/nrf9161 \
+            mcuboot/boot/zephyr
 
 #. Flash it onto the board as described in Option 1.
 
-#. Flash other Zephyr application to fill in slot0 e.g:
+#. Flash other Zephyr application using ``mcumgr``.
+
+.. tabs::
+
+   .. group-tab:: nRF52840
+
+      Build the blinky example with MCUboot support:
+
+      .. code-block:: console
+
+         west build \
+            -b ctcc/nrf52832 \
+            samples/basic/blinky \
+            -- \
+            -DCONFIG_BOOTLOADER_MCUBOOT=y \
+            -DCONFIG_MCUBOOT_SIGNATURE_KEY_FILE=\"path/to/mcuboot/boot/root-rsa-2048.pem\"
+
+   .. group-tab:: nRF9161
+
+      Build the blinky example with MCUboot support:
+
+      .. code-block:: console
+
+         west build \
+            -b ctcc/nrf9161 \
+            samples/basic/blinky \
+            -- \
+            -DCONFIG_BOOTLOADER_MCUBOOT=y
+            -DCONFIG_MCUBOOT_SIGNATURE_KEY_FILE=\"path/to/mcuboot/boot/root-rsa-2048.pem\"
+
+Flash produced image over USB port using ``mcumgr`` and reset target to boot into the image:
```

Comment:
no way can this broken tool that was deprecated ages ago be documented nor should it ever be used, if you want to document uploading using a tool then use one of the supported tools from https://docs.zephyrproject.org/latest/services/device_mgmt/mcumgr.html#tools-libraries

Reply:
In the [Overview](https://docs.zephyrproject.org/latest/services/device_mgmt/mcumgr.html#overview) paragraph in the MCUmgr the link driects to https://github.com/apache/mynewt-mcumgr where in the readme there is an instruction of installation of the mcumgr-cli that's why I used it, but you're right it's outdated. I used other client from the table and it works fine. I removed mcumgr-cli mentions and replaced it with links to the table of supported mcumgr clients. Thanks.